yarl/_url.py

```diff
@@ -551,7 +552,21 @@ def query(self):
         Empty value if URL has no query part.
 
         """
-        ret = MultiDict(parse_qsl(self.raw_query_string, keep_blank_values=True))
+        if (
+            (3, 6, 13) <= sys.version_info < (3, 7)
+            or (3, 7, 10) <= sys.version_info < (3, 8)
+            or (3, 8, 8) <= sys.version_info < (3, 9)
+            or sys.version_info >= (3, 9, 2)
```

Comment:
This module is not the right place for such conditionals. We should put `parse_qsl()` compat code into a `_compat` module and then import it here + use w/o conditionals.